ci/teams/rhtap-build/release.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  name: build-release-to-quay
```

in gitops file, you removed prefix...

```suggestion
  name: release-to-quay
```